Makefile

```diff
@@ -44,21 +48,31 @@ setup:
 get_thrift:
 	scripts/travis/get-thrift.sh
 
+# We want to remove `vendor` dir because thrift-gen tests don't work with it.
+# However, glide install even with --cache-gopath option leaves GOPATH at HEAD,
+# not at the desired versions from glide.lock, which are only applied to `vendor`
+# dir. So we move `vendor` to a temp dir and prepend it to GOPATH.
+# Note that glide itself is still executed against the original GOPATH.
 install:
-	glide --debug install --cache --cache-gopath
+	GOPATH=$(OLD_GOPATH) glide --debug install --cache --cache-gopath
+	[ -d $(VENDOR_PATH) ] && rm -rf $(VENDOR_PATH) || true
```

Comment:
`rm -rf` should return a successful exit code even if the directory doesn't exist. can you simplify this to just `rm -rf $(VENDOR_PATH)`